### PR TITLE
Migrate core/workspace_drag_surface_svg.js to goog.module syntax

### PR DESCRIPTION
**Pull request by gonfunko**
## The basics

- [x] I branched from `goog_module`
- [x] My pull request is against `goog_module`
- [x] My code follows the [style guide](
      https://developers.google.com/blockly/guides/modify/web/style-guide)
- [x] My code is presented in the form suggested in the [module
      conversion guide](https://github.com/google/blockly/issues/5026)
- [x] I have run `npm test`.

### Resolves

Part of #5026

### Proposed Changes

Converts `core/workspace_drag_surface_svg.js` to `goog.module` with ES6 `const`/`let`.

### CODE REVIEW

File: core/workspace_drag_surface_svg.js [hunk 1]
```diff
@@ -117,15 +117,17 @@ WorkspaceDragSurfaceSvg.prototype.getSurfaceTranslation = function() {
  */
 WorkspaceDragSurfaceSvg.prototype.clearAndHide = function(newSurface) {
   if (!newSurface) {
-    throw Error('Couldn\'t clear and hide the drag surface: missing ' +
+    throw Error(
+        'Couldn\'t clear and hide the drag surface: missing ' +
         'new surface.');
```
Comment (cpcallen) [inline core/workspace_drag_surface_svg.js:122]: Nit: I _think_ this will fit on a single line now.

File: core/workspace_drag_surface_svg.js [hunk 2]
```diff
@@ -117,15 +117,17 @@ WorkspaceDragSurfaceSvg.prototype.getSurfaceTranslation = function() {
...
   }
   const blockCanvas = /** @type {!Element} */ (this.SVG_.childNodes[0]);
   const bubbleCanvas = /** @type {!Element} */ (this.SVG_.childNodes[1]);
   if (!blockCanvas || !bubbleCanvas ||
       !dom.hasClass(blockCanvas, 'blocklyBlockCanvas') ||
       !dom.hasClass(bubbleCanvas, 'blocklyBubbleCanvas')) {
-    throw Error('Couldn\'t clear and hide the drag surface. ' +
+    throw Error(
+        'Couldn\'t clear and hide the drag surface. ' +
         'A node was missing.');
```
Comment (cpcallen) [inline core/workspace_drag_surface_svg.js:131]: Maybe here too?

**Review by cpcallen — APPROVED** [review]
Great—though it may be possible to further improve on clang's reformatting.

File: core/workspace_drag_surface_svg.js (hunk 1 above)
Comment (gonfunko) [reply core/workspace_drag_surface_svg.js]: Done!

File: core/workspace_drag_surface_svg.js (hunk 2 above)
Comment (gonfunko) [reply core/workspace_drag_surface_svg.js]: Done

**Pull request merged**